content/guides/userguides/accessing-metadata-via-tdr.md

```diff
@@ -0,0 +1,127 @@
+---
+date: "2021-12-25"
+description: "Overview of accessnig proejct metadata via the Terra Data Repository (TDR)."
+draft: false
+title: "Accessing Metadata via TDR"
+---
+
+# Accessing Metadata via TDR
+
+The [Terra Data Repository](https://support.terra.bio/hc/en-us/sections/4407099323675-Terra-Data-Repository)
+(TDR) offers a complete and authoritative source for the metadata made 
+available by the Data Portal data browser. This guide will walk you through the 
```

Comment:
Is "Data Portal data browser" really the correct way to say this? Capitalization and redundancy feel very awkward.

Reply:
The existing guide pages fluctuate a bit in the name with examples like:
- https://data.humancellatlas.org/guides
  - `The Data Portal Explore page ...`
  - `... top of the Data Portal.`
- https://data.humancellatlas.org/guides/quick-start-guide
  - `... using the Data Browser ...`
  - `... The Explore section of the Data Portal provides an interactive data browser ...`
  - `... selecting various facets in the Browser's Organ, Method, Donor ...`
- https://data.humancellatlas.org/guides/consumer-vignettes/export-to-terra
  - `... search results from the HCA Data Explorer to ...`
  - `... use the HCA Data Explorer to find ...`
  - `... the Data Explorer will process ...`
- https://data.humancellatlas.org/guides/consumer-vignettes/matrices
  - `... matrices available from the Data Portal.`
  - `... the "Matrices" column of the DCP Data Browser ...`

I believe the overall site is the `HCA Data Portal` and the `/explore` page is the `DCP Data Browser`, so I will use the latter.